docs/05_best-practices/04_data-design-and-migration.md

```diff
@@ -0,0 +1,41 @@
+# Data design and migration
+
+EOSIO based blockchains allow for smart contract code update, that is, you can upload a new version of the smart contract code easily however when it comes to data update and/or migration a few things need to be considered. The main structure for storing data in EOSIO based blockchains is the multi index table. Once a multi index table has been created with a first version of a smart contract, it has some limitations when it comes to changing its structure. Below we will go over a few possbile approaches which you can consider when you design your smart contract data and its migration. 
```

Comment:
`EOSIO based blockchains allow developers to easily update their smart contract code. However, a few things need to be considered when it comes to data update and/or migration. The main structure for storing data in EOSIO based blockchains is the multi index table.`